model_zoo/models/molecular_autoencoder_candle_pilot2/model_conv_molecular_autoencoder_pilot2.prototext

```diff
@@ -146,16 +164,17 @@ model {
   # FULLY_CONNECTED decode1
   #################
   layer {
+    parents: "relu_fc1"
     name: "d_fc1"
     data_layout: "data_parallel"
     fully_connected {
       num_neurons: 11264
-      weight_initialization: "glorot_uniform"
```

Comment:
Is this being removed because it is the default?

Reply:
This field is deprecated and doesn't have any effect. It's from before we had independent weight objects.